measuring_federate/measuring_federate.py

```diff
@@ -95,14 +100,15 @@ def __init__(self, config: MeasurementConfig, input_mapping, broker_config:Broke
             "publication", h.HELICS_DATA_TYPE_STRING, ""
         )
 
-        self.gaussian_variance = config.gaussian_variance
+        self.additive_noise_stddev = config.additive_noise_stddev
+        self.multiplicative_noise_stddev = config.multiplicative_noise_stddev
         self.measurement_file = config.measurement_file
-        self.random_percent = config.random_percent
 
     def transform(self, measurement_array: MeasurementArray, unique_ids):
         new_array = reindex(measurement_array, unique_ids)
         return apply(
-            lambda x: x + self.rng.normal(scale=np.sqrt(self.gaussian_variance)),
+            lambda x: (1 + self.rng.normal(scale=self.multiplicative_noise_stddev)) * x
```

Comment:
Why is there a '1+' in the expression? does it make comprehension of results different than before? If so, it should be documented in the release douments